### ISSUE TITLE
prop-types and sort-comp are throwing errors

### ISSUE THREAD

**Issue by tylermcginnis** [yannickcr/eslint-plugin-react#292]
When including `"react/prop-types": 2` and running the linter I get 

```
/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint-plugin-react/lib/util/Components.js:113
      return sourceCode.getText(node.parent.callee) === 'React.createClass';
                                           ^
TypeError: Cannot read property 'callee' of null
    at Object.context.react.isES5Component (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint-plugin-react/lib/util/Components.js:113:44)
    at Object.context.react.getParentES5Component (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint-plugin-react/lib/util/Components.js:183:35)
    at isPropTypesUsage (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint-plugin-react/lib/rules/prop-types.js:31:63)
    at EventEmitter.MemberExpression (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint-plugin-react/lib/rules/prop-types.js:561:11)
    at emitOne (events.js:82:20)
    at EventEmitter.emit (events.js:169:7)
    at NodeEventGenerator.enterNode (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint/lib/util/node-event-generator.js:42:22)
    at CommentEventGenerator.enterNode (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint/lib/util/comment-event-generator.js:98:23)
    at Controller.controller.traverse.enter (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint/lib/eslint.js:782:36)
    at Controller.__execute (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint/node_modules/estraverse/estraverse.js:397:31)
    at Controller.traverse (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint/node_modules/estraverse/estraverse.js:495:28)
```

in my console. 

Also,

When I run `"react/sort-comp": 2`

I get

```
/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint-plugin-react/lib/rules/sort-comp.js:307
var propertiesNames = properties.map(getPropertyName);
                                    ^
TypeError: Cannot read property 'map' of undefined
    at checkPropsOrder (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint-plugin-react/lib/rules/sort-comp.js:307:37)
    at EventEmitter.Program:exit (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint-plugin-react/lib/rules/sort-comp.js:353:9)
    at emitOne (events.js:82:20)
    at EventEmitter.emit (events.js:169:7)
    at NodeEventGenerator.leaveNode (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint/lib/util/node-event-generator.js:51:22)
    at CommentEventGenerator.leaveNode (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint/lib/util/comment-event-generator.js:111:23)
    at Controller.controller.traverse.leave (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint/lib/eslint.js:785:36)
    at Controller.__execute (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint/node_modules/estraverse/estraverse.js:397:31)
    at Controller.traverse (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint/node_modules/estraverse/estraverse.js:485:28)
    at EventEmitter.module.exports.api.verify (/Users/tyler/.nvm/versions/io.js/v2.5.0/lib/node_modules/eslint/lib/eslint.js:779:24)
```

Everything else seems to work great. I'm running "react-native": "0.13.0" and "eslint-plugin-react": "^3.7.0"

Thanks so much!

**Comment by yannickcr** [comment]
Duplicate of #286 and #285, these 2 issues where fixed in `v3.7.1`.

**Comment by tylermcginnis** [comment]
Ah sorry! I assumed I had the most recent version since I downloaded this morning. Thanks again!
